File: src/solutions/001/001_arvindch.py
#!/usr/bin/env python2.7

__author__ = 'Arvind Chembarpu'
__email__ = 'achembarpu@gmail.com'


def main():
    
    giv_lim = 1000
    req_sum = 0
    
    for n in xrange(giv_lim):
        if n % 3 == 0 or n % 5 == 0:
            req_sum += n
    
    return req_sum


if __name__ == '__main__':
    answer = main()
    print answer

File: src/java/solutions/001/_001_arvindch.java
// Author: Arvind Chembarpu
// Email: achembarpu@gmail.com

public class _001_arvindch {

	public static void main(String[] args) {
		int n, giv_lim = 1000, req_sum = 0;
		
		for(n=0; n<giv_lim; n++) {
			
			if(n%3==0 || n%5==0) {
				req_sum += n;
			}
		}
		
		System.out.println(req_sum);
	}

}
